### PR TITLE
Update Japanese translation (and more)

### PR DESCRIPTION
**Pull request by wtsnjp**
The first commit (16003e4c4944e668594121a6c2eb9ef9c177d8f0) includes most of the essential updates on the Japanese translation.

However, to make Japanese messages (and possibly those of other languages) more natural, it was necessary to use place holders instead of concatenating strings. So, I made a bit changes in the actual script and all translation files to make those compatible with the new place holders (ca710729f81c9f04e0632afdfe815233597331ef). I believe I successfully modified all things to work harmoniously, but if you think this change will make troubles, I will revert it.

### CODE REVIEW

**Comment by norbusan** [comment]
Should I commit this for you via svn? Merging here does not have any effect, as it will be overwritten immediately.

**Comment by wtsnjp** [comment]
Hi, I know you are busy now, but may I remind this once?

**Comment by norbusan** [comment]
Can you please update it, it is completely outdated.

**Comment by wtsnjp** [comment]
Ok, I will. Please let me know the best way to send the patch for you (maybe on GitHub or e-mail).

**Comment by norbusan** [comment]
Whatever is fine, patch or PR, I get the patch directly from the pull request ;-)

Thanks

**Comment by wtsnjp** [comment]
I think the PR is updated. Please kindly check and commit it to the svn repo.

File: texmf-dist/scripts/texlive/tlmgrgui.pl [hunk 1]
```diff
@@ -1367,7 +1367,7 @@ sub do_paper_settings {
       if (($p eq "context") && !defined($localtlpdb->get_package("bin-context"))) {
         next;
       }
-      $l{$p} = $back_config_pap->Label(-text => __("Default paper for") . " $p", -anchor => "w");
+      $l{$p} = $back_config_pap->Label(-text => __("Default paper for %s", $p), -anchor => "w");
       $m{$p} = $back_config_pap->Label(-textvariable => \$changedpaper{$p}, -anchor => "w");
```
Comment (norbusan) [inline texmf-dist/scripts/texlive/tlmgrgui.pl]: Is it on purpose that you change the Perl files, too?

**Comment by wtsnjp** [comment]
First, thanks for commiting the `ja.po`.

> Are you sure that you want all these changes?

Yes, I would like to. This is because some of current messages do not have enough place holders to make natural expressions in Japanese (and possibly in other languages.)

I believe I successfully changed all things (the perl code and corresponding translation texts with the new place holders) in harmony, but if it seems to make some troubles, I can leave somewhat unnatural Japanese translations.

**Comment by norbusan** [comment]
I included this into the main texlive, so it is now also already here. Closing this PR.

Thanks a lot!

**Pull request closed without merging**